### ISSUE TITLE
Custom Keyboards aren't working

### ISSUE THREAD

**Issue by nixterrimus** [mayoral/telegrammer#2]
This never shows customs keyboards-

``` ruby
reply_markup = Telegrammer::DataTypes::ReplyKeyboardMarkup.new(
  keyboard: [
    ["Option 1.1", "Option 1.2"],
    ["Option 2"],
    ["Option 3.1", "Option 3.2", "Option 3.3"]
  ],
  resize_keyboard: true,
)

bot.send_message(
  chat_id: chat_id,
  text: ">",
  reply_markup: reply_markup
)
```

I see `>` in my chat, but the Apple keyboard stays up.

Am I missing something?

**Comment by mayoral** [comment]
It seems that the Bot API needs to receive all parametes with values, so when the gem tried to send the reply_markup converted to JSON it was sending null values, in this case for the `one_time_keyboard` and `selective` parameters.

In 52247f0f845cdd639a6cb55447e0decffb54fe04 I've set up default values for the keyboard objects matching the default values of the official docs.

I've just released version 0.0.3 including this fix. Enjoy!
